docs/docs/howto/hardening.md

```diff
@@ -1,8 +1,19 @@
-# Security Considerations
+# Hardening
 
-Contrast ensures application integrity and provides secure means of communication and bootstrapping (see [security benefits](../basics/security-benefits.md)).
+This section provides guidance on writing secure applications and explains the trust boundaries developers need to be aware of when working with Contrast.
+
+## Applicability
+
+Recommended for all Contrast deployments.
+
+## Prerequisites
+
+A running Contrast deployment.
```

Comment:
I feel that the only real prerequisite here is the _intent_ to run a workload with Contrast _in prod_.